sympy/core/tests/test_logic.py

```diff
@@ -172,3 +173,7 @@ def test_formatting():
     raises(ValueError, lambda: S('a&b'))
     raises(ValueError, lambda: S('a|b'))
     raises(ValueError, lambda: S('! a'))
+
+
+def test_issue_16094():
```

Comment:
One substitution test will suffice, and that could be in test_basic.

Reply:
so, should I replace all the tests with a single test in test_basic?

Reply:
I think that test_basic would be the proper place for a `subs` test. There is no need for multiple copies in different files. They only repeat the same test.

Reply:
@divyanshu132 If you can figure out what variables that now can be unicode, feel free to add a test to that method where unicode characters/strings are used. It may not always be obvious though.